pkg/tso/global_allocator.go

```diff
@@ -200,7 +221,8 @@ func (gta *GlobalTSOAllocator) GenerateTSO(count uint32) (pdpb.Timestamp, error)
 		// we need to validate it first before we write it into every Local TSO Allocator's memory.
 		globalTSOResp = *estimatedMaxTSO
 		if err = gta.SyncMaxTS(ctx, dcLocationMap, &globalTSOResp, skipCheck); err != nil {
-			log.Error("global tso allocator synchronizes MaxTS failed", errs.ZapError(err))
+			log.Error("global tso allocator synchronizes MaxTS failed",
+				zap.Uint32("keyspace-group-id", gta.am.ksgID), errs.ZapError(err))
```

Comment:
I think we may need to remove the keyspace concept for the OP environment.

Reply:
done. removed for the time being. will add a work item to write logWrapper as discussed